src/Lumi/Components2/Button.purs

```diff
@@ -0,0 +1,200 @@
+module Lumi.Components2.Button where
+
+import Prelude
+
+import Data.Array as Array
+import Data.Char (fromCharCode)
+import Data.Foldable (fold)
+import Data.Maybe (Maybe)
+import Data.Nullable as Nullable
+import Data.String.CodeUnits (fromCharArray)
+import Effect (Effect)
+import Foreign.Object (fromHomogeneous)
+import Lumi.Components (LumiComponent', LumiComponentProps')
+import Lumi.Components.Size (Size(..))
+import Lumi.Styles.Button (ButtonKind(..), ButtonState(..))
+import Lumi.Styles.Button as Styles.Button
+import Prim.Row (class Union)
+import React.Basic.DOM as R
+import React.Basic.DOM.Events (capture_)
+import React.Basic.Emotion as E
+import React.Basic.Hooks (ReactComponent, component, useContext)
+import React.Basic.Hooks as React
+
+type CommonButtonProps rest =
+  ( accessibilityLabel :: Maybe String
+  , onPress :: Effect Unit
+  , size :: Size
+  , type :: String
```

Comment:
Do we want to keep using `String` props like this or is this just temporary or based on the old API?

Reply:
I was unsure so I left it for now. It's still relevant to the browser for things like `<button type="reset" />` or "submit", but it didn't seem like it should be tied into the style so I called that "kind".